cmd/proxy/Dockerfile

```diff
@@ -32,6 +32,9 @@ RUN chmod 644 /config/config.toml
 RUN apk add --update git git-lfs mercurial openssh-client subversion procps fossil tini && \
 	mkdir -p /usr/local/go
 
+ARG USER=athens
+RUN adduser -D -h /home/$USER $USER
```

Comment:
It might break k8s and docker-compose which provide external options to control non root access.

Reply:
k8s just lets you set the user id and group id to be run, much like docker does. Should you however need to use `.netrc` or `.gitconfig` you cannot without the user you are using having a home directory.

Reply:
Makes sense! Should we create non root group as well

Reply:
This is already done by `adduser`. The group is 1000 called `athens`.